lua/entities/gmod_wire_egp/lib/egplib/usefulfunctions.lua

```diff
@@ -258,6 +258,169 @@ function EGP:DrawLine( x, y, x2, y2, size )
 	end
 end
 
+function EGP:DrawPath( vertices, size, closed )
+	if false then
```

Comment:
Some dead code

Reply:
Yeah left than in to easily debug both versions. Will be removed in the final version.